python/cuML/cuml.pyx

```diff
@@ -6,3 +6,10 @@ include "linear_model/linear_regression.pyx"
 include "knn/knn_wrapper.pyx"
```

Comment:
This file no longer exists after the python reorganization and I'm sure it'd go away when you merge 0.6 into the PR, but I think this function would be good in a `utils.pyx` file (or some other logical name) inside a `utils` folder (where I'm planning to put more stuff), instead of in a generically named `cuml.pyx`.